boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -1,20 +1,38 @@
+#include <stdlib.h>
+#include <math.h>
+#include "App_SharedMacros.h"
 #include "states/App_AllStates.h"
 #include "states/App_InitState.h"
 #include "App_SetPeriodicCanSignals.h"
+#include "configs/App_EfficiencyLUT.h"
 
-// TODO: Have separate maximum torque requests for motoring and generating
 #define MAX_TORQUE_REQUEST_NM (90.0f)
+#define RPM_TO_RADS(rpm) ((rpm) * (float)M_PI / 30.0f)
```

Comment:
if some of these values are shared with the wheel speed sensor i suggest placing this macro in App_SharedMacros, and placing some #defines in App_SharedConstants

Reply:
M_PI shouldn't require a cast btw. There should be a floating pt version for PI if this M_PI is a double

Reply:
I don't see a floating point PI